File: Server/Server/Demo/logic/Player.cs
using System;
using YT;

public class Player {
	//id
	public string id = "";
	//指向ClientState
	public Connection state;
	//构造函数
	public Player(Connection state){
		this.state = state;
	}
	
	//数据库数据
	public PlayerData data;

	//发送信息
	public void Send(MsgBase msgBase){
		state.Send(msgBase);
	}

	

}

File: Server-Go/Demo/Player/Player.go
package Player

import (
	"ReaminHistory/YT"
	"google.golang.org/protobuf/runtime/protoiface"
)

type Player struct {
	//id
	Id string
	//指向ClientState
	Conv uint32
	//数据库数据
	Data *PlayerData
}

func NewPlayer(conv uint32 ) *Player {
	return &Player{Conv: conv}
}

//发送消息
func (player *Player) Send(msg protoiface.MessageV1) {
	YT.Clients[player.Conv].Send(msg)
}
